### ISSUE TITLE
Install pyenv-win to %ChocolateyInstall% instead of under USERPROFILE

### ISSUE THREAD

**Issue by davetapley** [bazzilic/ChocolateyPackages#16]
This would give an elegant solution to #15, whilst also helping with https://github.com/pyenv-win/pyenv-win/issues/321.

---

We presently install in `USERPROFILE`:
https://github.com/bazzilic/ChocolateyPackages/blob/3fdb41235a1693be04957007944abc72e3e5f002/pyenv-win/chocolateyInstall.ps1.template#L1-L3

But per [the docs](https://docs.chocolatey.org/en-us/faqs#portable-application-something-that-doesnt-require-a-system-install-to-use):
> A portable application is something that doesn't require a native installer to use. In other words, it is not “installed” on your system (where you can go to uninstall in the control panel). It also requires no administrative access for the package install.
> 
> Portable applications end up in the %ChocolateyInstall%/lib (i. e. C:\ProgramData\Chocolatey\lib) folder yes, but they get a "shim" to put them on the path of the machine. 

---

So, is there any objection to changing `-UnzipLocation` to be under `$env:ChocolateyInstall`?

**Comment by bazzilic** [comment]
Would you like to make a PR?

**Comment by davetapley** [comment]
@bazzilic I've got too much going right now to justify, but I will keep it in mind!

One thing to think: Should installing to `%ChocolateyInstall%` just become 'the default'?
And if so would that break `uninstall` from prior versions? 😬

**Comment by bazzilic** [comment]
Yes, it might break a couple of things, good catch. I will look into this when I have a chance.

**Comment by kirankotari** [comment]
@bazzilic Hey thanks for following up on it. If the pyenv paths are set correctly it might not break. 

By default we are recommending to use `%USERPROFILE%` but it has a freedom to install at any path. 
 - Chocolatey 
 - PowerShell 
these both automated installation tools are using default recommended path to keep simple and everyone in same page.

Example:- Lets say I have installed using git and one of team member installed using automated script (Chocolatey, PowerShell) and he has some issue/bug. it makes complex for other to validate/debug the issue/bug the team member is facing if we change the paths based on tools.
